### PR TITLE
Taxable Environment

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Add environemnt_ids and environments to taxonomies and TaxHost.

### CODE REVIEW

**Comment by timogoebel** [comment]
Is this still relevant?

**Comment by ezr-ondrej** [comment]
> Is this still relevant?

Depends, do we want to get Environment into the taxonomy tabs?

File: app/models/concerns/foreman_puppet/extensions/taxonomy.rb [hunk 1]
```diff
@@ -14,6 +14,14 @@ def dup
         new.environments = environments
         new
       end
+
+      def environment_ids do
```
Comment (timogoebel) [inline app/models/concerns/foreman_puppet/extensions/taxonomy.rb:18]:
```suggestion
      def environment_ids
```

?

**Comment by timogoebel** [comment]
> Depends, do we want to get Environment into the taxonomy tabs?

Well, I guess we want to keep the behavior before extraction. If I checked correctly, this functionality is still left in core. But I checked in a rush.

My question is: Do we need this in the 1.0 release?

**Comment by ezr-ondrej** [comment]
> My question is: Do we need this in the 1.0 release?

My understanding is no. It is regression from before extracted, but OTH no plugin entities are added to the Taxonomy edit :shrug:

**Comment by stejskalleos** [comment]
Closing the PR since there haven't been any updates for almost two years.

**Pull request closed without merging**